bin/ruby-build

```diff
@@ -643,8 +647,9 @@ build_package_ree_installer() {
   done
 
   # Work around install_useful_libraries crash with --dont-install-useful-gems
-  mkdir -p "$PREFIX_PATH/lib/ruby/gems/1.8/gems"
+  mkdir -p "$BUILT_PREFIX_PATH/lib/ruby/gems/1.8/gems"
 
+  # TODO: this is one PREFIX_PATH i'm not sure if it should be BUILT_ or not
   { ./installer --auto "$PREFIX_PATH" --dont-install-useful-gems $options $CONFIGURE_OPTS
```

Comment:
Not sure what to do about this, please advise. This is one of the functions that is well outside my usual paths with `ruby-build`.